va_explorer/va_analytics/dash_apps/va_dashboard.py

```diff
@@ -574,20 +573,70 @@ def get_metric_display_names(map_metrics):
     ],
 )
 
+# ============ VA data (loaded from database and shared across components) ========
+
+@app.callback(
+    Output(component_id="va_data", component_property="children"),
+    [
+        Input(component_id="timeframe", component_property="value"),
```

Comment:
Currently not using this. Should it be passed to load_va_data()?

Reply:
I included it because

1) I *think* callbacks need an input of some sort? Could be wrong about that
2) We may want to use that in the query at some point